### PR TITLE
feat(storybook): enable SVGR support for MDX stories

### PR DESCRIPTION
**Pull request by trumbitta**

### CODE REVIEW

File: packages/react/plugins/webpack.ts [hunk 1]
```diff
@@ -15,9 +15,9 @@ function getWebpackConfig(config: Configuration) {
     {
       test: /\.svg$/,
       oneOf: [
-        // If coming from JS/TS file, then transform into React component using SVGR.
+        // If coming from JS/TS or MDX file, then transform into React component using SVGR.
         {
-          issuer: /\.[jt]sx?$/,
+          issuer: /\.[jtm][sd]x?$/,
```
Comment (puku0x) [inline packages/react/plugins/webpack.ts:20]:
```suggestion
          issuer: /\.(js|ts|md)x?$/,
```

`.jd` `.td` `.ms` `.jdx` `.tdx` `.msx` should be ignored.

**Comment by puku0x** [comment]
`@nrwl/next` has `@svgr/webpack` loader too.
https://github.com/nrwl/nx/blob/3fa53857d7f7545ed9fbf6660b54d7115c0e6440/packages/next/src/utils/config.ts#L99-L109

File: packages/react/plugins/webpack.ts (hunk 1 above)
Comment (trumbitta) [reply packages/react/plugins/webpack.ts]: you're right: done

**Comment by trumbitta** [comment]
> `@nrwl/next` has `@svgr/webpack` loader too.
> 
> https://github.com/nrwl/nx/blob/3fa53857d7f7545ed9fbf6660b54d7115c0e6440/packages/next/src/utils/config.ts#L99-L109

But does it need support in MDX files? I have no experience with Next.js

**Comment by puku0x** [comment]
Never mind, there is nothing to do with `@nrwl/next`.

**Review by jaysoo — APPROVED** [review]

**Comment by trumbitta** [comment]
🎉

**Pull request merged**